motioneye/sendtelegram.py

```diff
@@ -1,3 +1,4 @@
+from __future__ import print_function
```

Comment:
Just to not forget: This is not required anymore with Python 3.

Reply:
But on Python 3 it throws a syntax error anyway, right? I.e. when CI is forced to test with `python3`, this becomes obsolete? However it's fine now as long as the code basis and CI is Python 2 compatible.